templates/basic/.github/workflows/size.yml

```
@@ -0,0 +1,15 @@
+name: "size"
+on:
+  pull_request:
+    branches:
+      - master
+jobs:
+  size:
+    runs-on: ubuntu-latest
+    env:
+      CI_JOB_NUMBER: 1
+    steps:
+      - uses: actions/checkout@v1
+      - uses: andresz1/size-limit-action@v1.3.0
```

```suggestion
      - uses: andresz1/size-limit-action@v1
```

Let's keep consistent with all the other actions and general NPM Semver practice and not pin to a specific patch version.